src/rect.rs

```diff
@@ -1,6 +1,9 @@
 use std::fmt;
 
-use mupdf_sys::*;
+use mupdf_sys::{
```

Comment:
Nitpick, but we use `*` imports in other places as well, and it's quite convenient as the functions are already "namespaced" from the C side, so `fz_*` is always clear to come from mupdf-sys.